hash/nes.xml

```diff
@@ -17574,10 +17574,11 @@ license:CC0
 	</software>
 
 	<software name="hyperolg" cloneof="hyperol">
-		<description>Hyper Olympic (Jpn, Gentaiban!)</description>
+		<description>Hyper Olympic (Japan, Genteiban!)</description>
```

Comment:
Why is “Genteiban” *after* the region like this?  If it’s part of the title, shouldn’t it be outside the parentheses?

Reply:
It's not technically part of the title. The in-game title screen is unchanged from the original release, and the box and cartridge are also unmodified originals with a sticker placed on them. [Here](https://stat.ameba.jp/user_images/20110507/21/retro-game/15/3e/j/t02200165_0640048011213532670.jpg?caw=800) is a picture of the cartridge where you can clearly see the original label on top.